boskos/janitor/janitor.go

```diff
@@ -44,42 +46,41 @@ func init() {
 func main() {
 	// Activate service account
 	flag.Parse()
+	extraJanitorFlags := flag.CommandLine.Args()
 
 	logrus.SetFormatter(&logrus.JSONFormatter{})
 	boskos := client.NewClient("Janitor", *boskosURL)
 	logrus.Info("Initialized boskos client!")
 
-	if *serviceAccount == "" {
-		logrus.Fatal("--service-account cannot be empty!")
-	}
-
 	if len(rTypes) == 0 {
 		logrus.Fatal("--resource-type must not be empty!")
 	}
 
-	cmd := exec.Command("gcloud", "auth", "activate-service-account", "--key-file="+*serviceAccount)
```

Comment:
pass down the service account into janitor?

Reply:
It's actually an optional flag now and should be passed by caller (e.g. kubernetes scenario), but looks like we are relying on GOOGLE_APPLICATION_CREDENTIALS everywhere.